docs/source/conf.py

```diff
@@ -169,7 +169,7 @@
 )
 
 # Change this to "always" before running in the doc's CI/CD server
-nbsphinx_execute = "never"
+# nbsphinx_execute = "never"
```

Comment:
This looks a bit sketchy, why not just delete this and the "Change this" comment above

Reply:
My bad, this was used in testing to speed up the build. I've removed it in the latest commit.